scripts/validate-charts

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env bash
+set -ex
```

Comment:
I would recommend running this without `set -x`, it is contributing a TON to the verbosity and length of the CI output.

```suggestion
set -e
```

Reply:
yes it was quite unreadable.
I added
`set +x` because `versions.sh` also sets `set -x`